package.json

```diff
@@ -9,7 +9,8 @@
     "test": "standard && (nyc -s tape test/*-test.js | faucet) && nyc report",
     "test-gc": "npx -n=--expose-gc tape test/{cleanup,iterator-gc,chained-batch-gc}*-test.js",
     "test-electron": "electron test/electron.js",
-    "test-prebuild": "cross-env PREBUILDS_ONLY=1 npm t",
+    "test-prebuild": "cross-env PREBUILDS_ONLY=1 tape test/*-test.js | faucet",
+    "test-prebuilds": "cross-env MSYS_NO_PATHCONV=1 bash ./scripts/test-prebuilds",
```

Comment:
The `MSYS_NO_PATHCONV=1` is a workaround for Windows, when using docker machine & git bash.